spec.bs

```diff
@@ -247,35 +260,55 @@ A user agent holds an <dfn>aggregatable report cache</dfn>, which is a [=list=]
 [=Implementation-defined=] values {#implementation-defined-values}
 ==================================================================
 
-<dfn>Max stored reports</dfn> is a positive integer that controls how many unsent reports can be queued waiting.
+<dfn>Maximum stored reports</dfn> is a positive integer that controls how many unsent reports can be queued waiting.
+
+<dfn>Maximum report contributions</dfn> is a positive integer that controls how many contributions can be present in a single report.
+
+<dfn>Minimum report delay</dfn> is a non-negative [=duration=] that controls the minimum delay to deliver an [=aggregatable report=]
 
-<dfn>Max report contributions</dfn> is a positive integer that controls how many contributions can be present in a single report.
+<dfn>Randomized report delay</dfn> is a positive [=duration=] that controls the random delay to deliver an [=aggregatable report=]. This delay is additional to the minimum delay.
 
 Issue: More
 
 Algorithms {#algorithms}
 ====================
 
+To <dfn>serialize an integer</dfn>, represent it as a [=string=] of the shortest possible decimal number.
 
-<!-- A {{WorkletGlobalScope}} |scope|'s <dfn algorithm for="PrivateAggregation">api identifier</dfn> is the result of:
-1. If |scope|'s [=worklet global scope type=] is {{FledgeWorkletGlobalScope}}, then return "<code>[=context type/fledge=]</code>".
-1. [=Assert=]: |scope|'s [=worklet global scope type=] is {{SharedStorageWorkletGlobalScope}}.
-1. Return "<code>[=context type/shared-storage=]</code>". -->
+Issue: This would ideally be replaced by a more descriptive algorithm in Infra. See infra/201
 
-To <dfn export algorithm for="PrivateAggregation">obtain a reporting endpoint</dfn> given an [=origin=] |reportingOrigin| and [=context type=] |api|, perform the following steps. They return a [=URL=].
+Scheduling reports {#scheduling-reports}
+----------------------------------------
+
+To perform the <dfn algorithm for="PrivateAggregation">report scheduling steps</dfn> with an [=environment settings object=] |context|, a [=context type=] |api| and a [=list=] of {{PAHistogramContribution}}s |contributions|:
+1. Let |reportingOrigin| be |context|'s [=environment settings object/origin=].
 1. [=Assert=]: |reportingOrigin| is a [=potentially trustworthy origin=].
-1. Let |path| be the [=string/concatenation=] of «"<code>.well-known/private-aggregation/report-</code>", |api|».
-1. Let |base| be the result on running the [=URL parser=] on the [=serialization of an origin|serialization=] of |reportingOrigin|.
-1. [=Assert=]: |base| is not failure.
-1. Let |result| be the result of running the [=URL parser=] on |path| with |base|.
-1. [=Assert=]: |result| is not failure.
-1. Return |result|.
+1. If the user agent's [=aggregatable report cache=] [=set/size=] is larger than [=maximum stored reports=], return.
```

Comment:
Note that ARA has removed this check in the spec. Shall we align given that this is not implemented as well for PAA?

Reply:
Yep, removed.